public/keymaps/m/montsinger_rebound_rev4_default.json

```
@@ -0,0 +1,45 @@
+{
+  "keyboard": "montsinger/rebound/rev4",
+  "keymap": "default",
+  "commit": "e56f4e8e752db58ef88be4e03ab35709c1e7eb1d",
```

Derived from the last commit to modify the keymaps from QMK's `master` branch, rather than any of your local branches.

```suggestion
  "commit": "b43bdc1c69bb5a57976a159bfc00c4ac1c7c7f8f",
```